frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -20,9 +19,26 @@
       "notes": "",
       "versus": ""
     },
+    "middleware-json": {
+      "setup_file": "setup-json",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "aspnetcore",
+      "language": "C#",
+      "orm": "Raw",
+      "platform": "NET",
```

Comment:
> .NET is a platform in the same way Java is
Java should not be classified as a platform. I realize that this classification has been murky in the past, and we are actively trying to clean it up with [this branch](https://github.com/TechEmpower/FrameworkBenchmarks/compare/cleanup-metadata).

Otherwise, I agree with your proposed setup of language/platform/framework/etc.